src/qt/res/README.md

```diff
@@ -0,0 +1,60 @@
+# Icon policy
+This document provides guidelines for contributing icons to Bitcoin Core.
+
+
+## Preparing Icons
+Both an icon source file, in Scalable Vector Graphics (SVG) format, and an optimized production file, in Portable Network Graphics (PNG) format, are required for 
+each icon.
+
+#### SVG Source File
+SVGs are used as source files because they can scale while retaining image quality, but are not used in production due to limited application support.

```

Comment:
the comma isn't actually needed here

```suggestion
SVGs are used as source files because they can scale while retaining image quality but are not used in production due to limited application support.

```